go.mod

```diff
@@ -27,15 +27,15 @@ require (
 	github.com/gorilla/handlers v1.4.0
 	github.com/gorilla/mux v1.7.0
 	github.com/gorilla/rpc v1.2.0+incompatible
-	github.com/hashicorp/vault v1.1.0
+	github.com/hashicorp/vault/api v1.0.5-0.20190730042357-746c0b111519
```

Comment:
Shouldn't that be `v1.0.5`?! (without date/commit-id)

Reply:
Thanks, updated to resolve.